python/test/test_entity_type_manager.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+
+def test_node_types(graph):
+    types = graph.node_types
+    assert {str(t) for t in types} == {
+        "kUnknownName",
```

Comment:
... I wonder why we included the `k` in the string version of this. Oh well. Not important.